<launch>
	<!-- joy node -->
	<node respawn="true" pkg="joy" type="joy_node" name="sputnik_joy_input">
		<param name="dev" type="string" value="/dev/input/js1" />
		<param name="deadzone" value="0.10" />
	</node>

	<!-- joystick node -->
	<param name="linear_axis" type="int" value="1" />
	<param name="angular_axis" type="int" value="0" />
	<param name="l_scale_max" type="double" value="1"/>
	<param name="a_scale_max" type="double" value="1"/>
	<node pkg="sputnik_joy" type="sputnik_joy_node" name="sputnik_joy" />
	
</launch>
